src/devtools/client/debugger/src/utils/editor/lineHitCounts.ts

```diff
@@ -1,26 +0,0 @@
-import { getUniqueHitCountsChunksForLines } from "ui/reducers/hitCounts";
-
-import { SourceEditor } from "./source-editor";
-
-export function calculateRangeChunksForVisibleLines(editor: SourceEditor) {
```

Comment:
:question: also curious where this went to

Reply:
`visibleLines` got moved into the `SourcesContext` – since they're specific to the currently selected source.

The `setVisibleLines` function updates them, and it gets called from the editor's scroll handler:
https://github.com/replayio/devtools/blob/3cf790059be8b7a70b40601c703d6d311df214d9/src/devtools/client/debugger/src/components/Editor/index.tsx#L271-L286

The reducer in that contexts buckets lines in a similar way to how `calculateRangeChunksForVisibleLines ` used to, I believe:
https://github.com/replayio/devtools/blob/1608dc4aa703514272f8c9e7d827f3ee87fcc722/packages/bvaughn-architecture-demo/src/contexts/SourcesContext.tsx#L146-L184